src/db/tests/batch.rs

```diff
@@ -242,3 +246,50 @@ async fn quota_test_append_batch() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_append_async_w_null() -> Result<()> {
+    let settings = crate::settings::test_settings();
+    if !settings.uses_spanner() {
+        dbg!("### Skipping test for mysql");
+        return Ok(());
+    }
+    let pool = db_pool(Some(settings)).await?;
+    let db = test_db(pool.as_ref()).await?;
+    let ttl = crate::db::util::ms_since_epoch() as u32;
+
+    let uid = 1;
+    let coll = "clients";
+    let first_bsos = vec![postbso(
+        "b0",
+        Some("payload 0"),
+        Some(10),
+        Some(ttl + 10_000),
+    )];
+    let new_batch = db.create_batch(cb(uid, coll, first_bsos)).await?;
+
+    // update the single bso twice, once changing
+    let updated_bsos = vec![
+        postbso("b0", None, Some(10), None),
```

Comment:
let's change the ttl here, to say `Some(20)` instead of the original value `Some(10)` (see below for the rest)